src/core/extras/ntt2.cpp

```diff
@@ -134,10 +134,8 @@ void test_NTT(const usint level, const usint nloop) {
     ILParams params1(m, q1, rootOfUnity1);
     auto x1p = std::make_shared<ILParams>(params1);
 
-    Poly::DugType dug1;
-    dug1.SetModulus(q1);  // random # generator to use
-
     // two vectors
+    Poly::DugType dug1;
```

Comment:
would like to make sure that dug1 doesn't need to be constructed with the modulus "q1".

Reply:
same